scripts/setup.sh

```diff
@@ -1,50 +1,141 @@
-# NOTE: the sudo invoker _HAS TO BE_ same as the auto-login user
-# Because we need to update that specific user's wayfire configs
-# Ensure making this abundantly clear in the install instuctions
-SITE=$(cat /etc/passwd | grep /$SUDO_USER: | cut -f6 -d:)
-cd $SITE
-
-# Update if there's a better way to install node than nodesource
-# Although, building (electron?) runtime binary can be an option
-[ $(which node) ] || curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
-
-# Install deps (apt update already handled by nodesource script)
-apt install -y git jq nodejs wtype
-
-# Clone the PiOSK repo, or (in case it exits) pull from upstream
-git clone https://github.com/debloper/piosk.git || git -C piosk pull
-
-# Set up wayfire autostart config to start up browser & switcher
-echo "[autostart]" >> $SITE/.config/wayfire.ini
-echo "browser = $SITE/piosk/scripts/browser.sh" >> $SITE/.config/wayfire.ini
-echo "switcher = bash $SITE/piosk/scripts/switcher.sh" >> $SITE/.config/wayfire.ini
-
-# If PiOSK config doesn't exist, try backup or use sample config
-if [ ! -f $SITE/piosk/config.json ]; then
-    if [ -f $SITE/piosk.config.bak ]; then
-        mv piosk.config.bak piosk/config.json
-    else
-        mv piosk/config.json.sample piosk/config.json
```

Comment:
This line has not been carried forward.  Currently a fresh install will fail due to a missing `config.json` file in /opt/piosk.